ldms/src/ldmsd/ldmsd_stream.c

```diff
@@ -156,6 +155,8 @@ int ldmsd_stream_publish(ldms_t xprt,
 	ldmsd_req_attr_t first_attr, attr, next_attr;
 	int rc;
 	size_t this_rec;
+	if (!data_len)
+		return 0;
```

Comment:
Hi @ballan, seems like this should be EINVAL?

Reply:
@tom95858  isn't the logical result of sending 0 bytes on a stream a successful no-op? seems like the function did what it was told.